src/core/generator/from_proto.rs

```diff
@@ -367,7 +369,7 @@ impl Context {
                 cfg_field.type_of = cfg_field.type_of.with_name(output_ty);
 
                 cfg_field.resolver = Some(Resolver::Grpc(Grpc {
-                    base_url: None,
+                    url: URL.to_string(),
```

Comment:
can't use a hardcoded url here. Take it from the user. Should throw a validation error if the user has not specified the url.